UnityGLTF/Assets/UnityGLTF/Shaders/UnityStandardInput.cginc

```diff
@@ -13,78 +13,78 @@
 //---------------------------------------
 // Directional lightmaps & Parallax require tangent space too
 #if (_NORMALMAP || DIRLIGHTMAP_COMBINED || _PARALLAXMAP)
-    #define _TANGENT_TO_WORLD 1
+	#define _TANGENT_TO_WORLD 1
 #endif
 
 #if (_DETAIL_MULX2 || _DETAIL_MUL || _DETAIL_ADD || _DETAIL_LERP)
-    #define _DETAIL 1
+	#define _DETAIL 1
 #endif
 
 //---------------------------------------
-half4       _Color;
-half        _Cutoff;
+half4	   _Color;
```

Comment:
I think a tool has done something strange here - this is an even weirder mix of tabs and spaces!